jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -92,10 +127,18 @@ public static String printSmartLocators(IBaseElement el) {
         }
     }
     public static JFunc1<IBaseElement, WebElement> SMART_SEARCH = el -> {
-        if (ELEMENT.useSmartSearch == FALSE ||
-            ELEMENT.useSmartSearch == ONLY_UI && el.base().locator == null ||
-            ELEMENT.useSmartSearch == UI_AND_ELEMENTS && el.base().locator == null && isInterface(el.getClass(), PageObject.class))
-            return null;
+        switch (ELEMENT.useSmartSearch) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)